Tests/ClientRuntimeTests/LoggingTests/SwiftLoggerTests.swift

```diff
@@ -0,0 +1,133 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+@testable import Smithy
+import ClientRuntime
+import Logging
+
+final class SwiftLoggerTests: XCTestCase {
```

Comment:
This test creates a `SwiftLogger` with a mock `LogHandler` injected (`LogHandler` is the entry point to `swift-log`; the mock is defined below the tests.)

A log statement is executed at every log level, and it is verified that the correct log params are received by the `LogHandler`.